09LessonNine/in-class-project/index.html

```diff
@@ -1,10 +1,17 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Intel</title>
+  <title>Gardening & Landscaping</title>
+  <link rel="stylesheet" href="css/css-reset.css"/>
   <link rel="stylesheet" href="css/style.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto+Condensed" rel="stylesheet">
 </head>
 <body>
-
+  <header>
+    <h4>Landscaping and More</h4>
+    <h1>We Craft Your Garden</h1>
+    <h3>check our gardening services and tools</h3>
```

Comment:
Don't forget to keep your fonts consistent - this `h3` is rendering differently from the `h1` and `h4` above